CMakeLists.txt

```diff
@@ -184,7 +184,10 @@ if(NOT DEFINED COMPILE_HIP_ATP_MARKER)
         set(COMPILE_HIP_ATP_MARKER 0)
     else()
         set(COMPILE_HIP_ATP_MARKER $ENV{COMPILE_HIP_ATP_MARKER})
+        message(WARNING "HIP Markers are deprecated, please use roctracer/rocTX marker APIs.")
     endif()
+else()
+    message(WARNING "HIP Markers are deprecated, please use roctracer/rocTX marker APIs.")
```

Comment:
Do we need the else? We would get the warning even if we didn't enable markers during HIP build.